frontend/src/components/GoalForm/ObjectiveTopics.js

```diff
@@ -15,27 +15,30 @@ export default function ObjectiveTopics({
   topics,
   onChangeTopics,
   status,
+  goalStatus,
   inputName,
   isOnReport,
   isOnApprovedReport,
   isLoading,
 }) {
   const initialSelection = useRef(topics.length);
 
-  const readOnly = status === 'Suspended' || (status === 'Not Started' && isOnReport);
+  const readOnly = useMemo(() => status === 'Suspended' || (status === 'Not Started' && isOnReport) || (status === 'Completed' && goalStatus === 'Closed'), [goalStatus, isOnReport, status]);
```

Comment:
Above you have '(status === 'Not Started' && isOnReport)' AND ' (status === 'In Progress' && isOnReport)'.

Do you need to check 'In Progress' with isOnReport (or I might be confused).

Reply:
This logic refects the mockup